src/AppInstallerCLITests/HttpClientHelper.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "TestCommon.h"
+#include "TestHandler.h"
+#include "AppInstallerErrors.h"
+#include <Rest/HttpClientHelper.h>
+
+using namespace AppInstaller::Repository::Rest;
+
+TEST_CASE("ExtractJsonResponse", "[RestSource]")

```

Comment:
>ExtractJsonResponse [](start = 11, length = 19)

For failed cases, it's better to name the test with "**Failed" or even better like "ExtractJsonResponseUnsupportedMimeType"